cli/cmd/replicaset.go

```diff
@@ -189,6 +191,27 @@ func newVShardCmd() *cobra.Command {
 	return cmd
 }
 
+// newRebootstrapCmd creates a "replicaset rebootstrap" command.
+func newRebootstrapCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "rebootstrap <APP_NAME:INSTANCE_NAME>",
+		DisableFlagsInUseLine: true,
+		Short:                 "Re-bootstraps an instance",
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdCtx.CommandName = cmd.Name()
+			err := modules.RunCmd(&cmdCtx, cmd.CommandPath(), &modulesInfo,
+				internalReplicasetRebootstrapModule, args)
+			util.HandleCmdErr(cmd, err)
+		},
+		Args: cobra.ExactArgs(1),
+	}
+
+	cmd.Flags().BoolVarP(&rebootstrapConfirmed, "yes", "y", false,
+		"Automatically confirm rebootstrap")
```

Comment:
Flags description starts with a lower-letter usually.
```suggestion
		"automatically confirm rebootstrap")
```